[2022-02-02 18:08:09,652] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: ex_1_data_ingestion_gcs_dag_v3.download_dataset_task scheduled__2019-03-01T00:00:00+00:00 [queued]>
[2022-02-02 18:08:09,664] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: ex_1_data_ingestion_gcs_dag_v3.download_dataset_task scheduled__2019-03-01T00:00:00+00:00 [queued]>
[2022-02-02 18:08:09,665] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-02 18:08:09,666] {taskinstance.py:1239} INFO - Starting attempt 1 of 1
[2022-02-02 18:08:09,667] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-02 18:08:09,679] {taskinstance.py:1259} INFO - Executing <Task(BashOperator): download_dataset_task> on 2019-03-01 00:00:00+00:00
[2022-02-02 18:08:09,686] {standard_task_runner.py:52} INFO - Started process 8375 to run task
[2022-02-02 18:08:09,691] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'ex_1_data_ingestion_gcs_dag_v3', 'download_dataset_task', 'scheduled__2019-03-01T00:00:00+00:00', '--job-id', '310', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion_hw_1_ex.py', '--cfg-path', '/tmp/tmpfakoh1o8', '--error-file', '/tmp/tmpcg6bf12v']
[2022-02-02 18:08:09,693] {standard_task_runner.py:77} INFO - Job 310: Subtask download_dataset_task
[2022-02-02 18:08:09,770] {logging_mixin.py:109} INFO - Running <TaskInstance: ex_1_data_ingestion_gcs_dag_v3.download_dataset_task scheduled__2019-03-01T00:00:00+00:00 [running]> on host 700aeb16bae0
[2022-02-02 18:08:09,814] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2022-02-02 18:08:09,842] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=ex_1_data_ingestion_gcs_dag_v3
AIRFLOW_CTX_TASK_ID=download_dataset_task
AIRFLOW_CTX_EXECUTION_DATE=2019-03-01T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2019-03-01T00:00:00+00:00
[2022-02-02 18:08:09,846] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-02-02 18:08:09,848] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'curl -sS https://s3.amazonaws.com/nyc-tlc/trip+data/yellow_tripdata_2019-03.csv > /opt/***/yellow_tripdata_2019-03.csv']
[2022-02-02 18:08:09,874] {subprocess.py:85} INFO - Output:
[2022-02-02 18:11:50,560] {subprocess.py:93} INFO - Command exited with return code 0
[2022-02-02 18:11:50,805] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=ex_1_data_ingestion_gcs_dag_v3, task_id=download_dataset_task, execution_date=20190301T000000, start_date=20220202T180809, end_date=20220202T181150
[2022-02-02 18:11:50,929] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-02 18:11:50,983] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2022-02-10 12:49:04,583] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: ex_1_data_ingestion_gcs_dag_v3.download_dataset_task scheduled__2019-03-01T00:00:00+00:00 [queued]>
[2022-02-10 12:49:04,604] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: ex_1_data_ingestion_gcs_dag_v3.download_dataset_task scheduled__2019-03-01T00:00:00+00:00 [queued]>
[2022-02-10 12:49:04,605] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-10 12:49:04,606] {taskinstance.py:1239} INFO - Starting attempt 1 of 1
[2022-02-10 12:49:04,607] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-10 12:49:04,623] {taskinstance.py:1259} INFO - Executing <Task(BashOperator): download_dataset_task> on 2019-03-01 00:00:00+00:00
[2022-02-10 12:49:04,635] {standard_task_runner.py:52} INFO - Started process 743 to run task
[2022-02-10 12:49:04,647] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'ex_1_data_ingestion_gcs_dag_v3', 'download_dataset_task', 'scheduled__2019-03-01T00:00:00+00:00', '--job-id', '666', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion_hw_1_ex.py', '--cfg-path', '/tmp/tmp49y0k4jq', '--error-file', '/tmp/tmpi28b985d']
[2022-02-10 12:49:04,650] {standard_task_runner.py:77} INFO - Job 666: Subtask download_dataset_task
[2022-02-10 12:49:04,728] {logging_mixin.py:109} INFO - Running <TaskInstance: ex_1_data_ingestion_gcs_dag_v3.download_dataset_task scheduled__2019-03-01T00:00:00+00:00 [running]> on host e30f201fae61
[2022-02-10 12:49:04,778] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2022-02-10 12:49:04,801] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=ex_1_data_ingestion_gcs_dag_v3
AIRFLOW_CTX_TASK_ID=download_dataset_task
AIRFLOW_CTX_EXECUTION_DATE=2019-03-01T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2019-03-01T00:00:00+00:00
[2022-02-10 12:49:04,803] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-02-10 12:49:04,805] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'curl -sS https://s3.amazonaws.com/nyc-tlc/trip+data/yellow_tripdata_2019-03.csv > /opt/***/yellow_tripdata_2019-03.csv']
[2022-02-10 12:49:04,826] {subprocess.py:85} INFO - Output:
[2022-02-10 12:50:51,073] {subprocess.py:93} INFO - Command exited with return code 0
[2022-02-10 12:50:51,147] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=ex_1_data_ingestion_gcs_dag_v3, task_id=download_dataset_task, execution_date=20190301T000000, start_date=20220210T124904, end_date=20220210T125051
[2022-02-10 12:50:51,240] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-10 12:50:51,307] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2022-02-10 15:32:05,216] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: ex_1_data_ingestion_gcs_dag_v3.download_dataset_task scheduled__2019-03-01T00:00:00+00:00 [queued]>
[2022-02-10 15:32:05,230] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: ex_1_data_ingestion_gcs_dag_v3.download_dataset_task scheduled__2019-03-01T00:00:00+00:00 [queued]>
[2022-02-10 15:32:05,231] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-10 15:32:05,232] {taskinstance.py:1239} INFO - Starting attempt 1 of 1
[2022-02-10 15:32:05,233] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-10 15:32:05,253] {taskinstance.py:1259} INFO - Executing <Task(BashOperator): download_dataset_task> on 2019-03-01 00:00:00+00:00
[2022-02-10 15:32:05,260] {standard_task_runner.py:52} INFO - Started process 231 to run task
[2022-02-10 15:32:05,271] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'ex_1_data_ingestion_gcs_dag_v3', 'download_dataset_task', 'scheduled__2019-03-01T00:00:00+00:00', '--job-id', '788', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion_hw_1_ex.py', '--cfg-path', '/tmp/tmp3ntv36jr', '--error-file', '/tmp/tmpa2obzlw4']
[2022-02-10 15:32:05,274] {standard_task_runner.py:77} INFO - Job 788: Subtask download_dataset_task
[2022-02-10 15:32:05,387] {logging_mixin.py:109} INFO - Running <TaskInstance: ex_1_data_ingestion_gcs_dag_v3.download_dataset_task scheduled__2019-03-01T00:00:00+00:00 [running]> on host 276a0ad90304
[2022-02-10 15:32:05,508] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2022-02-10 15:32:05,550] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=ex_1_data_ingestion_gcs_dag_v3
AIRFLOW_CTX_TASK_ID=download_dataset_task
AIRFLOW_CTX_EXECUTION_DATE=2019-03-01T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2019-03-01T00:00:00+00:00
[2022-02-10 15:32:05,552] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-02-10 15:32:05,553] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'curl -sS https://s3.amazonaws.com/nyc-tlc/trip+data/yellow_tripdata_2019-03.csv > /opt/***/yellow_tripdata_2019-03.csv']
[2022-02-10 15:32:05,598] {subprocess.py:85} INFO - Output:
[2022-02-10 15:32:56,302] {subprocess.py:93} INFO - Command exited with return code 0
[2022-02-10 15:32:56,343] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=ex_1_data_ingestion_gcs_dag_v3, task_id=download_dataset_task, execution_date=20190301T000000, start_date=20220210T153205, end_date=20220210T153256
[2022-02-10 15:32:56,385] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-10 15:32:56,439] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2022-02-10 16:30:06,150] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: ex_1_data_ingestion_gcs_dag_v3.download_dataset_task scheduled__2019-03-01T00:00:00+00:00 [queued]>
[2022-02-10 16:30:06,164] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: ex_1_data_ingestion_gcs_dag_v3.download_dataset_task scheduled__2019-03-01T00:00:00+00:00 [queued]>
[2022-02-10 16:30:06,165] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-10 16:30:06,167] {taskinstance.py:1239} INFO - Starting attempt 1 of 1
[2022-02-10 16:30:06,168] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-10 16:30:06,190] {taskinstance.py:1259} INFO - Executing <Task(BashOperator): download_dataset_task> on 2019-03-01 00:00:00+00:00
[2022-02-10 16:30:06,200] {standard_task_runner.py:52} INFO - Started process 181 to run task
[2022-02-10 16:30:06,207] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'ex_1_data_ingestion_gcs_dag_v3', 'download_dataset_task', 'scheduled__2019-03-01T00:00:00+00:00', '--job-id', '929', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion_hw_1_ex.py', '--cfg-path', '/tmp/tmpljee6z8s', '--error-file', '/tmp/tmpqbbr9fv5']
[2022-02-10 16:30:06,209] {standard_task_runner.py:77} INFO - Job 929: Subtask download_dataset_task
[2022-02-10 16:30:06,335] {logging_mixin.py:109} INFO - Running <TaskInstance: ex_1_data_ingestion_gcs_dag_v3.download_dataset_task scheduled__2019-03-01T00:00:00+00:00 [running]> on host 106a34111faf
[2022-02-10 16:30:06,439] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2022-02-10 16:30:06,491] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=ex_1_data_ingestion_gcs_dag_v3
AIRFLOW_CTX_TASK_ID=download_dataset_task
AIRFLOW_CTX_EXECUTION_DATE=2019-03-01T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2019-03-01T00:00:00+00:00
[2022-02-10 16:30:06,494] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-02-10 16:30:06,498] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'curl -sS https://s3.amazonaws.com/nyc-tlc/trip+data/yellow_tripdata_2019-03.csv > /opt/***/yellow_tripdata_2019-03.csv']
[2022-02-10 16:30:06,524] {subprocess.py:85} INFO - Output:
[2022-02-10 16:31:44,446] {subprocess.py:93} INFO - Command exited with return code 0
[2022-02-10 16:31:44,534] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=ex_1_data_ingestion_gcs_dag_v3, task_id=download_dataset_task, execution_date=20190301T000000, start_date=20220210T163006, end_date=20220210T163144
[2022-02-10 16:31:44,645] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-10 16:31:44,758] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
